app.html

```diff
@@ -1034,6 +1034,10 @@
 async function Loop() {
   while (true) {
     const preset = await Send("get_preset");
+    if (preset == -1){ //might happen after deletion of a preset
```

Comment:
Generally, if preset == -1, it means the lat preset in the list.
Not entirely sure if that's the right behavior in this particular case or not though.

Reply:
What I observed during deletion of a preset, is that the OS will select the 1st preset and announce its font. The WebAPP might not highlight the preset because of the -1. Therefore I made this (dirty?) fix after deletion

Reply:
But if it selects the first preset, why does it return -1?
Maybe this is a bug in ProffieOS itself?

Reply:
It only occurs very random. It is partly the same for highlighting the on/off buttons. Sometimes 'off' remains highlighted, although the saber is actually on. Seems to be some timing issues in the WebBLE. Effect is less for WebUSB.

Reply:
In that case, this is definitely the wrong way to fix the problem.
We need to figure out why this occurs and fix it instead of trying to patch the symptoms